src/collections/_documentation/platforms/php/index.md

```diff
@@ -324,10 +324,21 @@ the events, but report them as sent.
 use Sentry\ClientBuilder;
 use Sentry\Transport\NullTransport;
 use Sentry\State\Hub;
+use Sentry\Transport\TransportFactoryInterface;
 
-$transport = new NullTransport();
-$builder = ClientBuilder::create(['dsn' => '___PUBLIC_DSN___']);
-$builder->setTransport($transport);
+$options = ['dsn' => '___PUBLIC_DSN___'];
```

Comment:
I would just inline this variable since we won't reuse it besides in the `ClientBuilder::create` method